src/modeler/AutoRest.Swagger/Model/ValidationUtilities.cs

```diff
@@ -61,7 +70,7 @@ public static bool IsXmsPageableOrArrayResponseOperation(Operation op, ServiceDe
                     return false;
                 }
 
-                if (entity.Definitions[modelLink.StripDefinitionPath()].Properties?.Values?.Any(type => type.Type == DataType.Array)??false)
+                if (entity.Definitions[modelLink.StripDefinitionPath()].Properties?.Values?.Any(type => type.Type == DataType.Array) ?? false)
```

Comment:
I believe you meant to type || between the conditions instead of &&.  I think merging these 3 long conditions will just make the code look awkward. The current split seems to be clean and readable. Leaving it as such